ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/location/CaptureLocationTaskMapViewModelTest.kt

```diff
@@ -14,11 +29,16 @@ import javax.inject.Inject
 class CaptureLocationTaskMapViewModelTest : BaseHiltTest() {
 
   @Inject lateinit var captureLocationTaskMapViewModel: CaptureLocationTaskMapViewModel
-  @Inject lateinit var baseMapViewModel: BaseMapViewModel
 
   @Test
   fun testGetMapConfig() {
     assertThat(captureLocationTaskMapViewModel.mapConfig)
-      .isEqualTo(baseMapViewModel.mapConfig.copy(allowGestures = false))
+      .isEqualTo(MapConfig(showOfflineImagery = true, allowGestures = false))
+  }
+
+  @Test
+  fun testGetMapConfigWithNoOfflineImagery() {
+    assertThat(captureLocationTaskMapViewModel.mapConfig)
+      .isNotEqualTo(MapConfig(showOfflineImagery = false, allowGestures = false))
   }
```

Comment:
IIUC, this test is redundant since the above would fail if any regression happens.